MyCryptoHQ/MyCrypto
# Add simplex promo

Closes https://github.com/MyCryptoHQ/MyCrypto/issues/1785

### Description

Adds simplex promo for European users. Add small visual update for promos

### Changes

- Slight visual upgrade 
- Add simplex promo
	- the promo is shown based on your browser's time zone
	- UTC+0 to UTC+4 is the range where the simplex promo is shown, this covers all of europe

### Steps to Test

- Be European __OR__ fly to Europe __OR__ just change the range in the source code to confirm

### Screenshots
<img width="646" alt="screen shot 2018-05-24 at 5 28 08 am" src="https://user-images.githubusercontent.com/13836920/40477101-9664f742-5f13-11e8-951d-86057952188f.png">
<img width="654" alt="screen shot 2018-05-24 at 5 28 21 am" src="https://user-images.githubusercontent.com/13836920/40477099-9656df9a-5f13-11e8-9170-561c7989ff38.png">
<img width="656" alt="screen shot 2018-05-24 at 5 28 28 am" src="https://user-images.githubusercontent.com/13836920/40477098-964613ea-5f13-11e8-9d76-88be3b72c141.png">
<img width="653" alt="screen shot 2018-05-24 at 5 28 32 am" src="https://user-images.githubusercontent.com/13836920/40477097-96219c4a-5f13-11e8-8de1-308ac83d2910.png">
